scripts/validate_rpm_specs.py

```diff
@@ -17,16 +17,22 @@
 import argparse
 import os
 import sys
+import logging
 
 repo_root_dir = os.path.realpath(os.path.join(
     os.path.dirname(__file__), os.pardir))
 sys.path.insert(0, repo_root_dir)
 
 from lib import exception
+from lib import config
+from lib import log_helper
 from lib.utils import is_package_installed
 from lib.utils import recursive_glob
 from lib.utils import run_command
 
+log_helper.LogHelper()
```

Comment:
I think it's better to place this in the `__main__` section, like it's done in `host_os.py` to avoid having code at the module context level.